src/main.rs

```diff
@@ -162,10 +162,9 @@ fn print_changed_files(
             EXIT_ERROR.store(true, Ordering::Relaxed);
         }
 
-        // let current_file_path = file_path.to_str().unwrap().to_owned();
-        let current_file_path = file_path.display().to_string();
+        let current_file_path = file_path.to_str().unwrap_or_default();
```

Comment:
@ftonato Here `to_string` allocates a new string. And `unwrap` would cause RustyWind to panic if it was given a `OsStr` that couldn't be turned into a String.

Instead we use `to_str` here and `unwrap_or_default` will return an empty str if it can't be converted